tests/DistributedLocking/LockSet/DefaultLockSetTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+namespace Tests\DistributedLocking\LockSet;
```

Comment:
Tests in a different namespace? Means you have to have explicit `use` statements for the loaded classes.

Just making sure this is intentional. Is there a reason to do it this way? I can see where it's kind of useful for autocompletion and stuff to not get things mixed up. But...I always used `cmd+T` to make a new test, and it works best if it puts it in the same namespace.